src/vs/workbench/contrib/notebook/browser/view/renderers/cellRenderer.ts

```diff
@@ -366,7 +367,7 @@ export class CodeCellRenderer extends AbstractCellRenderer implements IListRende
 			outputShowMoreContainer,
 			editor,
 			templateDisposables,
-			elementDisposables: new DisposableStore(),
+			elementDisposables: this._register(new DisposableStore()),
```

Comment:
`elementDisposables` should be registered into the `templateDisposables` -- this has a similar leak as `renderTemplate` can be called an arbitrary number of times.